### PR TITLE
Add MSU Federal Credit Union

### PR DESCRIPTION
**Pull request by mckenn55**

### CODE REVIEW

**Review by brycx — CHANGES_REQUESTED** [review]
Thanks for the PR @mckenn55. I have some concerns:
- The docs say that:
> by choosing to receive a verification code sent via text message, email, or automated voice message.

So the `email: Yes` tag is missing.
- The docs need to explain _how to_ setup 2FA(the current link only says it's available). I couldn't find anything on their homepage, so if the how-to is only for registered customers, please provide a screenshot that documents the setup. Redact personal information, if any.

File: _data/banking.yml [hunk 1]
```diff
@@ -557,7 +557,14 @@ websites:
       twitter: meridiancu
       facebook: MeridianCreditUnion
       tfa: No
-
+
```
Comment (Carlgo11) [inline _data/banking.yml]: Could you remove these extra spaces?

File: _data/banking.yml [hunk 2]
```diff
@@ -557,7 +557,14 @@ websites:
       twitter: meridiancu
       facebook: MeridianCreditUnion
       tfa: No
-
+      
+    - name: Michigan State University Federal Credit Union
```
Comment (Carlgo11) [inline _data/banking.yml]: They call themselves MSU Federal Credit Union. Please change the name to that.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo you chose contains lots of text which looks distorted when shrunk down to 32x32 pixels.
I suggest you use this logo instead:
It contains their MSU characters but without the small text.
msufcu

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**